### ISSUE TITLE
MenuItem violates a11y rules

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#291]
<img width="1108" alt="image" src="https://github.com/element-hq/compound/assets/2403652/0e3a32ae-710e-4a6c-8a18-718af5376862">

**Comment by daniellekirkwood** [comment]
@t3chguy is this all menuitem s? i think you mentioned this in the bounds of the right panel?

**Comment by t3chguy** [comment]
@daniellekirkwood this is an issue with the Compound component, it applies to all usages of said component. Currently Element Web only uses Compound MenuItem in the RoomSummaryCard RightPanel
